app/routes/xiview_xi2.py

```
@@ -0,0 +1,304 @@
+from index import get_session
+from db_config_parser import get_xiview_base_url
+from xi2annotator.annotation import annotate_request
+
+xiview_xi2_data_router = APIRouter(include_in_schema=False)
+
+
+logger = logging.getLogger(__name__)
+
+
+@xiview_xi2_data_router.get('/get_peaklist', tags=["xiVIEW"])
+async def get_peaklist(id, sd_ref, upload_id):
```

Ruff (0.15.14)

[error] Function argument `id` is shadowing a Python builtin

(A002)